### PR TITLE
[4.0] Debug mode styling

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #14735

### Summary of Changes

This PR migrates the styling for debug mode from BS2 to BS4

screeny

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 93841e16ab377c79afe541a63b31a9f8a8ab0ab3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14739">issues.joomla.org/tracker/joomla-cms/14739</a>.</sub>

**Comment by C-Lodder** [comment]
@dgt41 - are you ok with the JS side of things?

File: media/plg_system_debug/js/debug.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Joomla = window.Joomla || {};
+ 
+(function( Joomla, document ) {
+	"use strict";
+
+	document.addEventListener('DOMContentLoaded', function() {
+		Joomla.toggleContainer = function(name)
+		{
+			var e = document.getElementById(name); // MooTools might not be available ;)
```
Comment (dgt41) [inline media/plg_system_debug/js/debug.js]: Plz remove that stupid comment

**Review by dgrammatiko — APPROVED** [review]

File: media/plg_system_debug/js/debug.js (hunk 1 above)
Comment (C-Lodder) [inline media/plg_system_debug/js/debug.js]: Oops.

**Pull request merged**